dev/grafana/provisioning/plugins/grafana-oncall-app-provisioning.yaml

```diff
@@ -5,5 +5,5 @@ apps:
     jsonData:
       stackId: 5
       orgId: 100
-      onCallApiUrl: http://oncall-dev-engine:8080
-      useBackendPlugin: false
+      onCallApiUrl: http://host.docker.internal:8080
```

Comment:
Fyi this one is not working by default if you run the project with Tilt

Reply:
Yeah I'll remove this before it gets merged.

Reply:
We might want to rethink how this file gets used as it gets loaded everytime the grafana instance is restarted.  This is another cause of confusion in helm and dev setups when it does not agree with config done from within the plugin itself.